neuraxle/steps/util.py

```diff
@@ -316,12 +318,89 @@ class DataShuffler:
 
 
 class NullPipelineSaver(PipelineSaver):
+    def load(self, pipeline: 'Pipeline', data_container: DataContainer) -> 'Pipeline':
+        return pipeline
+
     def can_load(self, pipeline: 'Pipeline', data_container: DataContainer) -> bool:
         return True
 
     def save(self, pipeline: 'Pipeline', data_container: DataContainer) -> 'Pipeline':
         return pipeline
 
+
+class ForEachDataInputs(TruncableSteps):
+    """
+    Truncable step that fits/transforms each step for each of the data inputs, and expected outputs.
+    """
```

Comment:
Plz not another thing in the util package, this package shouldn't even exist in the first place :sweat_smile: